python/tk_multi_workfiles/my_tasks/my_tasks_model.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, project, user, extra_display_fields, parent, bg_task_manager=
         """
         self.extra_display_fields = extra_display_fields or []
         filters = [["project", "is", project],
-                   ["task_assignees", "is", user]]
+                   ["task_assignees", "is", user],
```

Comment:
Instead of allowing people to simply take over the status filter, I think we should simply have the whole filter be an option in the yml file, just like we do for the entity tabs. This would allow people to do whatever they want.